remoteappmanager/docker/container.py

```diff
@@ -32,12 +32,22 @@ class Container(HasTraits):
     #: the id that will go in the URL of the container
     url_id = Unicode()
 
+    #: The absolute base prefix path for the container.
+    base_urlpath = Unicode(None, allow_none=True)
```

Comment:
I honestly don't have a clear answer to this problem. I see that a container should know its absolute path, but it is unable to do so when created. It's information that needs to be added later, because it's simply not available until later.